src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -905,6 +902,55 @@ namespace DataHeatBalance {
+        std::string Name;
+        std::string OSCMName;                       // OtherSideConditionsModel
+        int OSCMPtr;                                // OtherSideConditionsModel index
+        Real64 Porosity;                            // fraction of absorber plate [--]
+        Real64 LWEmitt;                             // Thermal Emissivity of Baffle Surface [dimensionless]
+        Real64 SolAbsorp;                           // Solar Absorbtivity of Baffle Surface [dimensionless]
+        Material::SurfaceRoughness BaffleRoughness; // surface roughness for exterior convection calcs.
+        Real64 PlenGapThick;                        // Depth of Plenum Behind Baffle [m]
+        int NumSurfs;                               // a single baffle can have multiple surfaces underneath it
+        Array1D_int SurfPtrs;                       // = 0  ! array of pointers for participating underlying surfaces
+        Real64 HdeltaNPL;                           // Height scale for Cavity buoyancy  [m]
+        Real64 AreaRatio;                           // Ratio of actual surface are to projected surface area [dimensionless]
+        Real64 Cv;                                  // volume-based effectiveness of openings for wind-driven vent when Passive
+        Real64 Cd;                                  // discharge coefficient of openings for buoyancy-driven vent when Passive
+        // data from elsewhere and calculated
+        Real64 ActualArea;  // Overall Area of Collect with surface corrugations.
+        Real64 ProjArea;    // Overall Area of Collector projected, as if flat [m2]
+        Vector Centroid;    // computed centroid
+        Real64 TAirCav;     // modeled drybulb temperature for air between baffle and wall [C]
+        Real64 Tbaffle;     // modeled surface temperature for baffle[C]
+        Real64 TairLast;    // Old Value for modeled drybulb temp of air between baffle and wall [C]
+        Real64 TbaffleLast; // Old value for modeled surface temperature for baffle [C]
+        Real64 HrPlen;      // Modeled radiation coef for OSCM [W/m2-C]
+        Real64 HcPlen;      // Modeled Convection coef for OSCM [W/m2-C]
+        Real64 MdotVent;    // air mass flow exchanging with ambient when passive.
+        Real64 Tilt;        // Tilt from area weighted average of underlying surfaces
+        Real64 Azimuth;     // Azimuth from area weighted average of underlying surfaces
+        Real64 QdotSource;  // Source/sink term
+        // reporting data
+        Real64 Isc;              // total incident solar on baffle [W]
+        Real64 PassiveACH;       // air changes per hour when passive [1/hr]
+        Real64 PassiveMdotVent;  // Total Nat Vent air change rate  [kg/s]
+        Real64 PassiveMdotWind;  // Nat Vent air change rate from Wind-driven [kg/s]
+        Real64 PassiveMdotTherm; // Nat. Vent air change rate from buoyancy-driven flow [kg/s]
+
+        // Default Constructor
+        ExtVentedCavityStruct()
```

Comment:
Missed opportunity to move the member initialization to the member declarations, but not a big deal.

Reply:
Oh, right, I can change the initialization to member declaration.

Reply:
You could, but I wouldn't worry about it if there's nothing else to change here.